src/snowflake/snowpark/modin/plugin/docstrings/resample.py

```diff
@@ -289,20 +289,121 @@ def ffill():
         2023-01-09  4.0  20.0
         """
 
-    def backfill():
```

Comment:
Removed from docstring since this is only used by `fillna` at the frontend (there actually isn't a `Resampler.backfill` pandas method). Ditto `pad`